packages/extraction/src/format-helper.js

```diff
@@ -0,0 +1,104 @@
+import { parse, serialize, FluentSerializer } from 'fluent-syntax';
+
+export function getAttributeName(attribute = {}) {
+  if (attribute.name) {
+    return attribute.name.name;
+  }
+}
+
+export function getAttributeValue(attribute = {}) {
+  if (attribute.value) {
+    return attribute.value.value;
+  }
+}
+
+function getEntryName(entry = {}) {
+  if (entry.id) {
```

Comment:
- [ ] is it possible to entry to be null? that wouldn't be caught by the `entry = {}`

Same with the above functions. Typescript could help our confidence here.

Reply:
https://github.com/udacity/fluent-react-utils/issues/22